src/disc-golf-events/DiscGolfEventDetails.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { Container, Typography, Paper, CircularProgress, Alert, Button } from '@mui/material';
+import config from '../config';
+
+interface DiscGolfEvent {
+    id: string;
+    tournamentTitle: string;
+    tournamentDate: string;
+    region: string;
+    pdga?: string;
+    externalLink?: string;
+}
+
+const DiscGolfEventDetails = () => {
+    const { id } = useParams<{ id: string }>();
+    const navigate = useNavigate();
+    const [event, setEvent] = useState<DiscGolfEvent | null>(null);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState<string>('');
+
+    const fetchEventDetails = useCallback(() => {
+        if (!id) {
+            setError('No tournament ID');
+            setLoading(false);
```

Comment:
I think we don't need this line. When starting to fetch event details it's almost certain, that spinner is not displayed anyway.